docs/serverless-v2/01-02-serverless-v2.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Architecture
+---
+
+Serverless v2 replies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Tekton](https://github.com/tektoncd/pipeline) as a pipeline for creating and running on-cluster processes. See how these and other resources process a lambda within a Kyma cluster:
+
+![Serverless v2 architecture](./assets/serverless-v2-architecture.svg)
```

Comment:
![image](https://user-images.githubusercontent.com/17271979/74136279-08ca5980-4bee-11ea-8f08-02892409bbee.png)
Does it imply tekton creates local registry for each function?

Reply:
and as per some @michal-hudy 's comment - change kservice to ksvc

Reply:
`kservice` is also in CRD ;) anyway just use one convention in the whole doc.

Reply:
@aerfio I will point to the Docker image directly here as you are right that can be quite confusing

Reply:
ok, I will change `KService` to `ksvc`

Reply:
As agreed with @michal-hudy , I will leave kservice